<!-- Home Section -->
<section id="home">
    <div class="col-lg-12 text-center">
        <h2 class="section-heading">SUMO</h2>
    </div>

    <div class="container">
        <div class="row">
            <div class="col-md-12">
                Welcome to the SUMO group! Our collective laboratory works
                on problems of computational biology while leveraging
                methods from bioinformatics, visualization, and image
                analysis. Our group has engineered several workflows for
                processing and analyzing images for problems in cancer,
                neuro, and wound biology and immunology. We have developed
                algorithms that have been adapted for exploration in the
                laboratory at the microscopic scale (confocal, brightfield,
                histology, etc.) and in radiology clinics (diffusion tensor
                MR, fMRI, etc.).  Our algorithms are derived from the fields
                of computer vision and machine learning. In addition, we are
                also developing tools for processing sequence data (micro
                array, RNA-seq, CHiP-seq). Kun Huang is also a Co- director
                of the Bioinformatics Share Resource facility in the
                Comprehensive Cancer Center. This combined focus of image
                informatics and bio informatics is serving us well and we
                collaborate with several groups on campus and elsewhere.
            </div>
        </div>
    </div>
</section>

<section id="projects">
    <div class="col-lg-12 text-center">
        <h2 class="section-heading">Projects</h2>
    </div>

    <div id="igpse" class="modal-content">
        <div class="container">
            <div class="row">
                <div class="col-lg-8 col-lg-offset-2">
                    <div class="modal-body">
                        <!-- Project Details Go Here -->
                        <h2>iGPSe</h2>

                        <p class="item-intro text-muted">A visual analytic
                        system for integrative genomic based cancer patient
                        stratification</p>

                        <p class="text-justify">We present a visual analytic
                           system called <a
                               href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-203">Interactive
                               Genomics Patient Stratification explorer</a>
                           (iGPSe) which significantly reduces the computing
                           burden for biomedical researchers in the process
                           of exploring complicated integrative genomics
                           data. Our system integrates unsupervised
                           clustering with graph and parallel sets
                           visualization and allows direct comparison of
                           clinical outcomes via survival analysis. Using a
                           breast cancer dataset obtained from the The
                           Cancer Genome Atlas (TCGA) project, we are able
                           to quickly explore different combinations of gene
                           expression (mRNA) and microRNA features and
                           identify potential combined markers for survival
                           prediction.</p>

                        <p class="text-justify">Cancers are highly
                        heterogeneous with different subtypes. These
                        subtypes often possess different genetic variants,
                        present different pathological phenotypes, and most
                        importantly, show various clinical outcomes such as
                        varied prognosis and response to treatment and
                        likelihood for recurrence and metastasis. Recently,
                        integrative genomics (or panomics) approaches are
                        often adopted with the goal of combining multiple
                        types of omics data to identify integrative
                        biomarkers for stratification of patients into
                        groups with different clinical outcomes. In this
                        paper we present a visual analytic system called
                        Interactive Genomics Patient Stratification explorer
                        (iGPSe) which significantly reduces the computing
                        burden for biomedical researchers in the process of
                        exploring complicated integrative genomics data. Our
                        system integrates unsupervised clustering with graph
                        and parallel sets visualization and allows direct
                        comparison of clinical outcomes via survival
                        analysis. Using a breast cancer dataset obtained
                        from the The Cancer Genome Atlas (TCGA) project, we
                        are able to quickly explore different combinations
                        of gene expression (mRNA) and microRNA features and
                        identify potential combined markers for survival
                        prediction.  Visualization plays an important role
                        in the process of stratifying given population
                        patients. Visual tools allowed for the selection of
                        possibly features across various datasets for the
                        given patient population. We essentially made a case
                        for visualization for a very important problem in
                        translational informatics.</p>

                        <p>
                            <iframe width="420" height="315"
                                src="https://www.youtube.com/embed/j8tCxOLjJhg"
                                frameborder="0" allowfullscreen>
                            </iframe>
                        </p>
                    </div>
                </div>
            </div>
        </div>
    </div>

    <!-- Portfolio Modal 2 -->
    <div id="graphie" class="modal-content">
        <div class="container">
            <div class="row">
                <div class="col-lg-8 col-lg-offset-2">
                    <div class="modal-body">
                        <h2>GRAPHIE</h2>

                        <p class="item-intro text-muted">Graph based
                        histology image explorer</p>

                        <p class="text-justify">We introduce <a
                           href="https://kb.osu.edu/dspace/handle/1811/73536">Graph
                           based
                           Hisportfolio/startup-framework-preview.pngtology
                           Image Explorer (GRAPHIE)</a>-a visual analytics
                       tool to explore, annotate and discover potential
                       relationships in histology image collections within a
                       biologically relevant context. The design of GRAPHIE
                       is guided by domain experts' requirements and
                       well-known InfoVis mantras. By representing each
                       image with informative features and then subsequently
                       visualizing the image collection with a graph,
                       GRAPHIE allows users to effectively explore the image
                       collection. The features were designed to capture
                       localized morphological properties in the given
                       tissue specimen. More importantly, users can perform
                       feature selection in an interactive way to improve
                       the visualization of the image collection and the
                       overall annotation process. Finally, the annotation
                       allows for a better prospective examination of
                       datasets as demonstrated in the users study. Thus,
                       our design of GRAPHIE allows for the users to
                       navigate and explore large collections of histology
                       image datasets.</p>

                        <p class="text-justify">Histology images comprise
                        one of the important sources of knowledge for
                        phenotyping studies in systems biology. However, the
                        annotation and analyses of histological data have
                        remained a manual, subjective and relatively
                        low-throughput process. We introduce Graph based
                        Histology Image Explorer (GRAPHIE)-a visual
                        analytics tool to explore, annotate and discover
                        potential relationships in histology image
                        collections within a biologically relevant context.
                        The design of GRAPHIE is guided by domain experts'
                        requirements and well-known InfoVis mantras. By
                        representing each image with informative features
                        and then subsequently visualizing the image
                        collection with a graph, GRAPHIE allows users to
                        effectively explore the image collection. The
                        features were designed to capture localized
                        morphological properties in the given tissue
                        specimen. More importantly, users can perform
                        feature selection in an interactive way to improve
                        the visualization of the image collection and the
                        overall annotation process. Finally, the annotation
                        allows for a better prospective examination of
                        datasets as demonstrated in the users study. Thus,
                        our design of GRAPHIE allows for the users to
                        navigate and explore large collections of histology
                        image datasets. We demonstrated the usefulness of
                        our visual analytics approach through two case
                        studies. Both of the cases showed efficient
                        annotation and analysis of histology image
                        collection.</p>

                        <p>
                            <iframe width="560" height="315"
                                src="https://www.youtube.com/embed/2mYZ0EijtKY"
                                frameborder="0" allowfullscreen>
                            </iframe>
                        </p>
                    </div>
                </div>
            </div>
        </div>
    </div>
</section>

<!-- Team Section -->
<section id="people" class="bg-light-gray">
    <div class="container">
        <div class="row">
            <div class="col-lg-12 text-center">
                <h2 class="section-heading">People</h2>
                <!--<h3 class="section-subheading text-muted">Lorem ipsum dolor sit amet consectetur.</h3>-->
            </div>
        </div>
        <div class="row">
            <div class="col-sm-4">
                <div class="team-member">
                    <img src="img/people/haoding.png" class="img-responsive img-circle" alt="">
                    <h4>Hao Ding</h4>
                    <p class="text-muted">Ph.D. Candidate in Computer Science &amp; Engineering</p>
                    <ul class="list-unstyled">
                        <li>The Ohio State University</li>
                        <li><i>tel:</i> 614-xxx-xxxx</li>
                        <li><i>email</i>: <a href="mailto:ding.96@osu.edu">ding.96@osu.edu</a></li>
                    </ul>
                </div>
            </div>
            <div class="col-sm-4">
                <div class="team-member">
                    <img src="img/people/kunhuang.png" class="img-responsive img-circle" alt="">
                    <h4>Kun Huang, Ph.D.</h4>
                    <p class="text-muted">Associate Professor, College of Medicine</p>
                    <ul class="list-unstyled">
                        <li>The Ohio State University</li>
                        <li><i>tel:</i> 614-292-5607</li>
                        <li><i>email</i>: <a href="mailto:kun.huang@osumc.edu">kun.huang@osumc.edu</a></li>
                    </ul>
                </div>
            </div>
            <div class="col-sm-4">
                <div class="team-member">
                    <img src="img/people/raghumachiraju.png" class="img-responsive img-circle" alt="">
                    <h4>Raghu Machiraju, Ph.D.</h4>
                    <p class="text-muted">Professor, College of Engineering and College of Medicine</p>
                    <ul class="list-unstyled">
                        <li>The Ohio State University</li>
                        <li><i>tel:</i> 614-2926730</li>
                        <li><i>email</i>: <a href="mailto:machiraju.1@osu.edu">machiraju.1@osu.edu</a></li>
                    </ul>
                </div>
            </div>
        </div>
    </div>
</section>

<section id="papers">
    <div class="container">
        <div class="row">
            <div class="col-lg-12 text-center">
                <h2 class="section-heading">Papers</h2>
            </div>
        </div>

        <div class="row">
            <ol>
                <li>iGPSe: A Visual Analytic System for Integrative Genomic
                    Based Cancer Patient Stratification . Ding H, Wang C,
                    Huang K, Machiraju R. International Symposium on
                    Biological Data Visualization (BioVis) 2014. <a
                        href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-203">[web]</a></li>

                <li>GRAPHIE: Graph based Histology Image Explorer. Ding H,
                    Wang C, Huang K, Machiraju R.  International Symposium
                    on Biological Data Visualization (BioVis) 2015. <a
                        href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-16-S11-S10">[web]</a></li>
            </ol>
        </div>
    </div>
</section>

<footer>
    <div class="container">
        <div class="row">
            <div class="col-md-offset-7 col-md-5 text-right">
                <div class="copyright">Copyright &copy; The Ohio State University 2016</div>
                <div class="copyright">All Rights Reserved</div>
            </div>
        </div>
    </div>
</footer>
